Specs/Number/Chinese/NumberModel.json

```diff
@@ -4230,5 +4230,47 @@
         }
       }
     ]
+  },
+  {
+    "Input": "一万零十二",
```

Comment:
Are these three new spec cases all situations? Isn't there any false positive we want to make sure doesn't happen?

Reply:
Yes, please add more real examples which are more complex and with more different entities.
Also just like @tellarin's comment, please add more false positive cases.